samples/ClassroomLabs/Scripts/BringImageToSharedImageGallery/Readme.md

```diff
@@ -0,0 +1,31 @@
+# Introduction
+This script is used to bring a custom image from an Azure virtual machine (VM) to shared image gallery.
+
+## Notes
+- The script must be executed using administrator privileges.
+
+# Directions
+1. Open a PowerShell window.  Make sure that the window notes it is running under administrator privileges.
+2. Run `BringImageToSharedImageGallery.ps1`.
+3. When prompted, enter information about where the source VM resides and the shared image gallery where the custom image will be created.  Here are some helpful tips:
```

Comment:
Only thing maybe to add is that the user can pass in all the necessary parameters (if they want), or just run the script and they'll be prompted for anything that's missing.